src/tools/debugger/BinaryWebSocketHandler.java

```diff
@@ -0,0 +1,44 @@
+package tools.debugger;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.CompletableFuture;
+
+import org.java_websocket.WebSocket;
+import org.java_websocket.handshake.ClientHandshake;
+import org.java_websocket.server.WebSocketServer;
+
+public class BinaryWebSocketHandler extends WebSocketServer{
+  private static final int NUM_THREADS = 1;
+  private final CompletableFuture<WebSocket> connection;
+
+  public BinaryWebSocketHandler(final InetSocketAddress address) {
+    super(address, NUM_THREADS);
+    connection = new CompletableFuture<>();
+  }
+
+  @Override
+  public void onClose(final WebSocket conn, final int arg1, final String arg2, final boolean arg3) {
+    // TODO Auto-generated method stub
```

Comment:
instead of the auto generated comment, for empty operations, I would prefer if you either leave the body blank, or have something like `// no-op`, or better, just remove the method if it wasn't abstract.